app/lib/actions/katello/repository/clone_to_version.rb

```diff
@@ -38,6 +41,13 @@ def plan(repositories, content_view_version, incremental = false)
             end
           end
         end
+
+        def extract_rpm_filenames(repos_units, repo_label)
+          if repos_units.present?
+            repo_units = repos_units.detect { |r| r[:repo_label] == repo_label }
```

Comment:
This is to double check that the passed in repo_label is present within the units?